test/reliability.jl

```diff
@@ -7,52 +7,71 @@ using LinearAlgebra, Random, Test
+    @test reliability_X.dropped[1] ≈ 0.75
+    @test reliability_X.dropped[2] ≈ 0.7605633802816901
+    @test reliability_X.dropped[3] ≈ 0.7714285714285715
+    @test reliability_X.dropped[4] ≈ 0.782608695652174
 
     # testing Rational
     cov_rational = cov_X .// 1
-    reliability_rational = crombach_alpha(cov_rational)
+    reliability_rational = crombachalpha(cov_rational)
     @test reliability_rational isa Reliability{Rational{Int}}
     @test reliability_rational.alpha == 48 // 59
-    @test reliability_rational.dropped[1].second == 3 // 4
-    @test reliability_rational.dropped[2].second == 54 // 71
-    @test reliability_rational.dropped[3].second == 27 // 35
-    @test reliability_rational.dropped[4].second == 18 // 23
+    @test reliability_rational.dropped[1] == 3 // 4
+    @test reliability_rational.dropped[2] == 54 // 71
+    @test reliability_rational.dropped[3] == 27 // 35
+    @test reliability_rational.dropped[4] == 18 // 23
 
     # testing BigFloat
     cov_bigfloat = BigFloat.(cov_X)
-    reliability_bigfloat = crombach_alpha(cov_bigfloat)
+    reliability_bigfloat = crombachalpha(cov_bigfloat)
     @test reliability_bigfloat isa Reliability{BigFloat}
     @test reliability_bigfloat.alpha ≈ 0.8135593220338981
-    @test reliability_bigfloat.dropped[1].second ≈ 0.75
-    @test reliability_bigfloat.dropped[2].second ≈ 0.7605633802816901
-    @test reliability_bigfloat.dropped[3].second ≈ 0.7714285714285715
-    @test reliability_bigfloat.dropped[4].second ≈ 0.782608695652174
+    @test reliability_bigfloat.dropped[1] ≈ 0.75
+    @test reliability_bigfloat.dropped[2] ≈ 0.7605633802816901
+    @test reliability_bigfloat.dropped[3] ≈ 0.7714285714285715
+    @test reliability_bigfloat.dropped[4] ≈ 0.782608695652174
 
     # testing corner cases
-    @test_throws MethodError crombach_alpha([1.0, 2.0])
+    @test_throws MethodError crombachalpha([1.0, 2.0])
     cov_k2 = [10 6;
               6 11]
-    reliability_k2 = crombach_alpha(cov_k2)
+    reliability_k2 = crombachalpha(cov_k2)
     @test reliability_k2.alpha ≈ 0.7272727272727273
     @test isempty(reliability_k2.dropped)
 
+    # testing when Matrix is not positive-definite
+    cov_not_pos = [-1 1;
+                   -1 1]
+    @test_throws ArgumentError crombachalpha(cov_not_pos)
+
+    # testing with a zero
+    cov_zero = [1 2;
+                0 1]
+    @test_throws ArgumentError crombachalpha(cov_not_pos)
+
+    # testing with one column
+    cov_k1 = reshape([1, 2], 2, 1)
+    @test_throws ArgumentError crombachalpha(cov_k1)
+
+    # testing with Missing
+    cov_missing = [1 2;
+                   missing 1]
+    @test_throws MethodError crombachalpha(cov_missing)
+
+
     # testing Base.show
-    reliability_X = crombach_alpha(cov_X)
+    reliability_X = crombachalpha(cov_X)
     io = IOBuffer()
     show(io, reliability_X)
     str = String(take!(io))
     @test str == """
         Reliability for all items: 0.8136
-
```

Comment:
Sorry, I didn't mean to remove this empty line in the output. Just add `\n` at the end of the string passed to `@printf` to restore it.

Reply:
> Thanks. Just a few more comments.
> 
> Can you also add a reference to the function in the docs so that it's included in the manual? Maybe a new section under Scalar Statistics?

I am finishing it up the `.md` file for documentation. I also noticed a huge mistake. It's Cronbach not Crombach! (https://en.wikipedia.org/wiki/Cronbach%27s_alpha). I will change the function naming accordingly...